### PR TITLE
Add information about monitoring service on VATZ

### LINKED ISSUE #353: Write overall node exporter
## Checklist

- [ ] New Feature for the Service/Plugin (Vatz)
- [ ] Enhancement (Vatz)
- [x] others(etc. e.g, documentation, project policy management)

---

### PR DESCRIPTION
**Pull request by gnongs**
### 1. Type of change

Please delete options that are not relevant.

- [ ] New feature
- [X] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [ ] others (anything other than above)

---

**Related:** # (issue)
closes #353
**Summary**

We need to inform VATZ supported metrics service.

So I added the information so that I can know the contents through the diagram.

---

image

### CODE REVIEW

File: docs/design.md [hunk 1]
```diff
@@ -62,18 +62,23 @@ Plugins that allow you to perform followings per protocols
...
+The most validator teams have trouble managing logs from running nodes due to log's varieties.<br/>
+**VATZ**'s monitoring service is designed to find a way to manage all logs from nodes efficiently with minimum cost.<br/>
 
-We are targeting for followings:
+**AS-IS** <br/>
+![monitoring-as-is](../docs/images/monitoring_as_is.png)
 
-> 1. Manage **VATZ** with Dashboard  (2022-Q3)
-> 2. Unified Log exporter (2023-Q2)
+Currnetly VATZ supported to send metics like liveness of plugins for promethes. Metrics may be added in the future.
```
Comment (rootwarp) [inline docs/design.md:72]: Maybe. 
- metics => metrics
- promethes -> Prometheus

Comment (gnongs) [reply docs/design.md]: Thanks @rootwarp.

Fixed typo - 487fdb6b84d6bfe1aeaecc9c894d63b4b063fdcc

File: docs/design.md [hunk 2]
```diff
@@ -62,18 +62,23 @@ Plugins that allow you to perform followings per protocols
...
+The most validator teams have trouble managing logs from running nodes due to log's varieties.<br/>
+**VATZ**'s monitoring service is designed to find a way to manage all logs from nodes efficiently with minimum cost.<br/>
 
-We are targeting for followings:
+**AS-IS** <br/>
+![monitoring-as-is](../docs/images/monitoring_as_is.png)
 
-> 1. Manage **VATZ** with Dashboard  (2022-Q3)
-> 2. Unified Log exporter (2023-Q2)
+Currnetly VATZ supported to send metrics like liveness of plugins for promethues. Metrics may be added in the future.
```
Comment (xellos00) [inline docs/design.md:72]: I prefer to update this sentence as 

VATZ currently supports sending metrics for followings for Prometheus: 
- `VATZ` up 
-  `VATZ` plugins liveness 

More metrics will be available in the future.

**Review by rootwarp — APPROVED** [review]
LGTM

File: docs/design.md (hunk 2 above)
Comment (gnongs) [reply docs/design.md]: @xellos00 updated it!
- Commit: 009e7c186cf3b4c77a352a1ee17165c707dd7c33

**Review by xellos00 — APPROVED** [review]

**Pull request merged**